backstop/tests/color-picker.html

```diff
@@ -0,0 +1,190 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>color-picker | iTwinUI</title>
```

Comment:
Why lower case?

Reply:
Oh didn't notice that, it was just the title it gave when using yarn createComponent, but I'll fix it

Reply:
Also change in line 29